ui/src/components/ContextInfoContent.vue

```diff
@@ -1,7 +1,9 @@
 <template>
     <div class="wrapper">
         <div class="title">
-            <h2>{{ props.title }}</h2>
+            <slot name="title">
+                <h2>{{ props.title }}</h2>
+            </slot>
```

Comment:
This way, you don't have to update the CSS (and expect the calling component to know that there is an h2 with title in it)

```suggestion
            <slot name="title"/>
            <h2>{{ props.title }}</h2>
```